cmake/configure/configure_20_trilinos.cmake

```diff
@@ -297,13 +307,13 @@ macro(feature_trilinos_find_external var)
           STATUS
           "MueLu was found but is not usable through Epetra! Disabling MueLu support."
           )
-        set(DEAL_II_TRILINOS_WITH_MUELU OFF)
+        set(TRILINOS_WITH_MUELU OFF)
       endif()
     endif()
 
     # the only thing we use from SEACAS right now is ExodusII, so just check
     # that it works
-    if(${DEAL_II_TRILINOS_WITH_SEACAS})
+    if(${TRILINOS_WITH_SEACAS})
```

Comment:
Why do you use ``if(${...})`` here and below but not above on line 278? I assume this is redundant.

Reply:
Beats me. I just ran a search and replace on `DEAL_II_TRILINOS` and I chose to not alter the script too much in order to avoid mistakes (sans some whitespace fixes further down below where I couldn't resist).